dataactbroker/handlers/fileHandler.py

```diff
@@ -133,6 +133,28 @@ def getErrorReportURLsForSubmission(self, isWarning = False):
             # Unexpected exception, this is a 500 server error
             return JsonResponse.error(e,StatusCode.INTERNAL_ERROR)
 
+    def getSignedURLForSubmissionFile(self):
```

Comment:
I wasn't sure about function names. I thought maybe leaving function names camelCase and make variable names underscored. We only discussed variable names specifically in the channel discussion so I figured it'd be a good way to differentiate them. Let me know if you disagree